stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -110,6 +114,11 @@ public class ChannelListViewModel(
         return channelMutes.value.any { cid == it.channel.cid }
     }
 
+    private var queryChannelsState: QueryChannelsState? = null
+
+    private val CHANNEL_LIMIT = 30
```

Comment:
This is not really a constant (as it has no const keyword) and it allocated on each constructor call.

But it's okay for now, I'll create a ticket and the compose team will take care of it.

Reply:
Yes, better to refactor by compose team.